specs/sb/api.md

```diff
@@ -364,6 +461,45 @@ A web-friendly display name is camel-cased with spaces and punctuation supported
         "bullets": [{
           "content": "40 concurrent connections"
         }]
+      },
+      "schemas": {
+        "service_instance": {
+          "parameters": {
+            "required": [
+              "fake_required_property"
+            ],
+            "properties": {
+              "fake_required_property": {
+                "type": "string"
+              }
+            }
+          },
+          "outputs": {
+            "properties": {
+              "fake_output": {
+                "type": "string"
+              }
+            }
+          }
+        },
+        "service_binding": {
+          "credentials": {
+            "properties": {
```

Comment:
In current form "credentials" entry looks like a required input parameter which used during a service binding. If this is an output, then it should be inside "outputs" entry just like it is done in "service_instance" example.